src/data/threads.js

```diff
@@ -291,6 +292,14 @@ async function createNewThreadForUser(user, opts = {}) {
       infoHeader += `\n\nThis user has **${userLogCount}** previous modmail threads. Use \`${config.prefix}logs\` to see them.`;
     }
 
+    //BYCOP
+    const row = await knex("notes")
+    .where("user_id", user.id)
+    .first();
+    if (row !== undefined && row.note !== "undefined") {
+      infoHeader += `\n\nNote :  **${row.note}**.`;
```

Comment:
In this and all other user-side text outputs you seem to have an extra space in front of each `:`, it might be better to remove that extra space.

Reply:
Thanks i'll fix it